src/components/Nav/Dropdown.tsx

```diff
@@ -1,203 +1,151 @@
-          {children}
-        </Box>
-      </ListItem>
-    </DropdownContext.Provider>
-  )
-}
-
-interface IItemProp {
-  children?: React.ReactNode
-  isLast?: boolean
-}
-
-const Item: React.FC<IItemProp> = ({ children, isLast = false, ...rest }) => {
-  const context = useContext(DropdownContext)
-
-  return (
-    <ListItem
-      {...rest}
-      onClick={() => context?.close()}
-      onKeyDown={(e) => context?.tabInteractionHandler(e, isLast)}
-      m={0}
-      color="inherit"
-      _hover={{
-        bg: "dropdownBackgroundHover",
-        color: "text",
-      }}
-    >
-      {children}
+    <ListItem m={0}>
+      <ChakraMenu isLazy placement="bottom">
+        {({ isOpen }) => (
+          <>
+            <MenuButton
+              dir={direction}
+              borderRadius="base"
+              px="3"
+              py="0.5"
+              _focusVisible={{
+                outline: "4px solid",
+                outlineColor: "primary.hover",
+                color: "primary.base",
+              }}
+              _hover={{
+                color: "primary.base",
+              }}
+              _active={{
+                bg: "primary.lowContrast",
+              }}
+              _expanded={{
+                bg: "background.highlight",
+              }}
+              sx={{
+                "& > span": {
+                  display: "inline-flex",
+                  alignItems: "center",
+                },
+              }}
+            >
+              {section.text}
+              <Icon
+                as={MdExpandMore}
+                fontSize="2xl"
+                color="currentcolor"
+                transform={isOpen ? "rotate(180deg)" : undefined}
+              />
+            </MenuButton>
+            <MenuList
+              border="1px"
+              borderColor="dropdownBorder"
+              borderRadius="none"
+              p={4}
+              sx={{ "--menu-bg": "colors.dropdownBackground" }}
+              // Override the dark styles from the default theme
+              _dark={{}}
+              rootProps={{
+                top: "4 !important",
+              }}
+            >
+              {children}
+            </MenuList>
+          </>
+        )}
+      </ChakraMenu>
     </ListItem>
   )
 }
 
-interface ITitleProps {
-  children?: React.ReactNode
-}
+const Item = (props: Pick<LinkProps, "to" | "children">) => (
+  <MenuItem
+    color="text200"
+    display="block"
+    p="2"
+    _hover={{
+      bg: "primary.hover",
+      color: "background.base",
+    }}
+    _active={{
+      bg: "primary.lowContrast",
+      color: "primary.highContrast",
+    }}
+    _focusVisible={{
+      outline: "4px solid",
+      outlineColor: "primary.hover",
+      outlineOffset: "-1px",
+      color: "primary.base",
+      bg: "none",
+    }}
+  >
+    <BaseLink
+      isPartiallyActive={false}
+      color="inherit"
+      textDecor="none"
+      _hover={{ textDecor: "none" }}
+      {...props}
+    />
```

Comment:
Having `BaseLink` as a child is creating invalid html since `MenuItem` is rendering a button.

Do you think we can do `<MenuLink as={BaseLink}>` here?

Reply:
Ah, you mean `MenuItem`?

I must have overlooked that issue. `MenuItem` should be fine here to consume `BaseLink` via `as`. Will check on that asap!

Reply:
Oh yes haha, `MenuItem` it is xD